janus_core/calculations/md.py

```diff
@@ -518,7 +518,7 @@ def get_log_stats(self) -> str:
             density = 0.0
             pressure_tensor = np.zeros(6)
 
-        log_stats = (
+        stats = (
```

Comment:
I think this might actually be better expressed as something like:
```python
stats = " ".join(f"{step:10d}",
                 f"{real_time.total_seconds():.3f}",
                 f"{time:13.2f}",
                 f"{e_pot:.8e}", ...
```
or
```
stats = " ".join(format(step, "10d"),
                 format(real_time.total_seconds(), ".3f"),
                 format(time, "13.2f"),
                 format(e_pot, "8e"), ...
```
But I'm not certain and not for this PR.

It does mean that for `pressure_tensor` you could do:
```python
*map(lambda x: format(x, "8e"), pressure_tensor)
```
I don't know.